WordPress/Classes/ViewRelated/Reader/ReaderSiteStreamHeader.swift

```diff
@@ -89,12 +90,21 @@ fileprivate func > <T: Comparable>(lhs: T?, rhs: T?) -> Bool {
         }
     }
 
-    @objc func configureHeaderImage(_ siteBlavatar: String?) {
+    @objc func configureHeaderImage(_ siteBlavatar: String?, isWPForTeams: Bool = false) {
```

Comment:
I don't see this method called outside the class, so it could be private and probably doesn't need to be `@objc`.